kt/kt.go

```diff
@@ -270,13 +270,13 @@ func init() {
 // because for some operations, we care about the order
 // and since we only care about direct key lookup in a few
 // cases where the sets are small, we can amortize the cost of the map
-type kv struct {
-	key   string
-	value []byte
+type Kv struct {
```

Comment:
`KV` by the acronym naming convention.

Needs better doc, the comment should probably be detached from the exported object.